docs/visualization/visualize_unit_responses.ipynb

```diff
@@ -65,10 +65,23 @@
   },
```

Comment:
<p>try to rerun this without an error?</p><p> </p><p>As a note: I make sure to completely rerun all cells before committing and making sure they run without errors so they were run exactly once and in order.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>